app/controllers/forest_liana/application_controller.rb

```diff
@@ -69,7 +65,12 @@ def authenticate_user_from_jwt
 
           @jwt_decoded_token = JWT.decode(token, ForestLiana.auth_secret, true,
             { algorithm: 'HS256', leeway: 30 }).try(:first)
-          @rendering_id = @jwt_decoded_token['data']['relationships']['renderings']['data'][0]['id']
+
+          if @jwt_decoded_token['data']
+            raise ForestLiana::Errors::HTTP401Error.new("Deprecated token format")
```

Comment:
[Wording] Choose a consistent sentence (with express lianas) to explain why the user is logged out.
"Deprecated token format" is a bit cryptic.
"Your token format is invalid, please login again." is probably better, ask for help for the wording if you need.